plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/thrift/ThriftHiveMetastoreClient.java

```diff
@@ -177,6 +185,55 @@ public Database getDatabase(String dbName)
         return client.getDatabase(dbName);
     }
 
+    @Override
+    public List<SchemaTableName> getAllTables()
+            throws TException
+    {
+        return alternativeCall(
+                exception -> !isUnknownMethodExceptionalResponse(exception),
+                chosenGetAllTablesAlternative,
+                () -> parseTableMetadata(client.getTableMeta("*", "*", ImmutableList.of())),
```

Comment:
If we pass an empty List of tableTypes will it select all types of tables or none ?

Reply:
all

Reply:
sure